File: app/src/main/java/me/lukeforit/spaceofaday/data/source/cache/SpaceDatabase.java
package me.lukeforit.spaceofaday.data.source.cache;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {ApodEntity.class}, version = 1)
public abstract class SpaceDatabase extends RoomDatabase {
    public static String DATABASE_NAME = "space-db";

    public abstract ApodDao apodDao();
}

File: app/src/main/java/me/lukeforit/spaceofaday/data/source/cache/SpaceDatabase.kt
package me.lukeforit.spaceofaday.data.source.cache

import androidx.room.Database
import androidx.room.RoomDatabase

const val DATABASE_NAME = "space-db"

@Database(entities = arrayOf(ApodEntity::class), version = 1)
abstract class SpaceDatabase : RoomDatabase() {

    abstract fun apodDao(): ApodDao
}
